www_src/pages/project/project.jsx

```diff
@@ -437,6 +437,30 @@ var Project = React.createClass({
     }
   },
 
+  setDestination: function () {
+    // var pageUrl = `/projects/${this.state.params.project}/pages/${this.state.routeData.pageID}`;
+
+    api({
+      method: 'patch',
+      uri: `/users/1/projects/${this.state.params.project}/pages/${this.state.routeData.pageID}/elements/${this.state.routeData.elementID}`,
+      json: {
+        attributes: {
```

Comment:
yeah, unfortunately you're going to have to pass along the rest of the attributes with `routeData`. Alternatively you could pass the chosen page *back* to the editor in `routeData`, instead of just triggering `goBack`, but i think the former solution would be a lot easier.